spec/requests/api/v1/users/sign_up_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'User acount creation', type: :request do
+  subject { post api_v1_user_registration_path, params: user_params, as: :json }
+
+  context 'when attributes are good' do
+    let(:user_email) { Faker::Internet.email }
+    let(:user_params) do
+      { email: user_email, password: Faker::Internet.password }
+    end
+
+    it 'creates an user account' do
+      expect { subject }.to change { User.count }.by(1)
+      expect(response).to have_http_status(:created)
+      expect(User.find_by(email: user_email)).to_not be_nil
+    end
```

Comment:
Remember to try to have only one expectation per example, I would also not add the `expect(User.find_by(email: user_email)).to_not be_nil` case
```ruby
    it 'returns created' do
      expect(response).to have_http_status(:created)
    end
    
    it 'creates the user' do
      expect { subject }.to change { User.count }.by(1)
    end
```